qcodes/instrument_drivers/rigol/DG1062.py

```diff
@@ -290,7 +304,32 @@ def _set_waveform_params(self, **params_dict: Dict) ->None:
         string += ",".join(
             ["{:7e}".format(params_dict[param]) for param in param_names])
         self.parent.write_raw(string)
+        
+    def _get_duty_cycle(self):
```

Comment:
what's the type of the return value?`float`? if so, please, add a type annotation like this:
```suggestion
    def _get_duty_cycle(self) -> float:
```